web/src/pages/Home.tsx

```
@@ -80,8 +80,20 @@ export default function Home() {
               </p>
-              <p className="mt-1 text-sm text-yellow-400">
-                ⚠️ 알림을 받으려면 먼저 <a href="https://t.me/chart_pattern_alimi_bot" target="_blank" className="text-blue-500 hover:underline">텔레그램 봇(@chart_pattern_alimi_bot)</a>과 대화를 시작해주세요 (/start 명령어)
+              <div className="mt-2 p-3 bg-blue-900/20 border border-blue-500/30 rounded-lg">
+                <p className="text-sm text-blue-300 font-medium mb-2">
+                  📱 Chat ID 확인 방법:
+                </p>
+                <ol className="text-sm text-blue-200 space-y-1 ml-4 list-decimal">
+                  <li>
+                    <a href="https://t.me/chart_pattern_alimi_bot" target="_blank" className="text-blue-400 hover:underline font-medium">텔레그램 봇</a>과 대화를 시작하세요
+                  </li>
+                  <li>봇에게 <code className="bg-blue-900 px-2 py-0.5 rounded text-blue-100">/start</code> 또는 <code className="bg-blue-900 px-2 py-0.5 rounded text-blue-100">/myid</code> 또는 <code className="bg-blue-900 px-2 py-0.5 rounded text-blue-100">/start</code> 명령어를 보내세요</li>
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)